js/apps/admin-ui/src/realm-settings/user-profile/attribute/AddTranslationsDialog.tsx

```diff
@@ -116,11 +116,32 @@ export const AddTranslationsDialog = ({
   }, [combinedLocales, translationKey, setValue]);
 
   const handleOk = () => {
-    const formData = getValues();
-    onTranslationsAdded(formData);
+    const formData = getValues() as Translations;
```

Comment:
Looks like this compiles the same without a type-cast, so let's remove it.
```suggestion
    const formData = getValues();
```